GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts

```diff
@@ -251,7 +251,9 @@ namespace gdjs {
         // Render all the layers then.
         // TODO: replace by a loop like in 3D?
         pixiRenderer.backgroundColor = this._runtimeScene.getBackgroundColor();
-        pixiRenderer.render(this._pixiContainer);
+        pixiRenderer.render(this._pixiContainer, {
+          clear: this._runtimeScene.clearCanvas,
```

Comment:
We could probably extend this to the 3D rendering

Reply:
I've never tried to use 3D, but it seems already covered here:
https://github.com/4ian/GDevelop/blob/master/GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts#L138

Reply:
It would unfortunately not work out of the box, because the 3D rendering is doing rendering layers by layers and the first layer is doing a `clear`, which should be avoided if this flag is set :)

Reply:
Got it. @4ian, so should I handle this case for 3D as well or the PR can be merged as is?